app/helpers/application_helper/button/new_host_aggregate.rb

```diff
@@ -1,5 +1,13 @@
 class ApplicationHelper::Button::NewHostAggregate < ApplicationHelper::Button::ButtonNewDiscover
+  def supports_button_action?
+    @supported_button_action ||= begin
+      filtered_providers = Rbac::Filterer.filtered(ManageIQ::Providers::CloudManager)
+      filtered_providers.empty? ? false : filtered_providers.all? { |ems| ems.supports?(:create_host_aggregate) }
```

Comment:
Why is this `filtered_providers.all? { |ems| ems.supports?(:create_host_aggregate) }` ?

Shouldn't this be `filtered_providers.any? { |ems| ems.supports?(:create_host_aggregate) }` ?